sdks/typescript/package.json

```diff
@@ -35,30 +39,33 @@
     "lint": "eslint . --ext .ts"
   },
   "dependencies": {
-    "@google-cloud/pubsub": "^2.19.4",
-    "@grpc/grpc-js": "~1.4.6",
-    "@protobuf-ts/grpc-transport": "^2.1.0",
-    "@protobuf-ts/plugin": "^2.1.0",
-    "bson": "^4.6.0",
-    "capture-console": "^1.0.1",
+    "@google-cloud/pubsub": "^4.1.0",
+    "@grpc/grpc-js": "~1.9.13",
+    "@protobuf-ts/grpc-transport": "^2.9.3",
+    "@protobuf-ts/plugin": "^2.9.3",
+    "@types/node": "^20.10.6",
+    "bson": "^6.2.0",
+    "capture-console": "^1.0.2",
     "chai": "^4.3.4",
-    "date-fns": "^2.28.0",
+    "date-fns": "^3.0.6",
     "fast-deep-equal": "^3.1.3",
     "find-git-root": "^1.0.4",
-    "long": "^4.0.0",
-    "protobufjs": "~6.11.3",
+    "long": "^5.2.3",
+    "protobufjs": "~7.2.5",
     "queue-typescript": "^1.0.1",
-    "serialize-closures": "^0.2.7",
-    "ts-closure-transform": "^0.1.7",
-    "ttypescript": "^1.5.15",
-    "uuid": "^8.3.2"
+    "serialize-closures": "^0.3.0",
+    "ts-closure-transform": "^0.2.0",
+    "typescript": "5.3",
+    "uuid": "^9.0.1"
   },
   "main": "./dist/src/apache_beam/index.js",
   "exports": {
     ".": "./dist/src/apache_beam/index.js",
     "./io": "./dist/src/apache_beam/io/index.js",
     "./runners": "./dist/src/apache_beam/index.js",
     "./transforms": "./dist/src/apache_beam/transforms/index.js",
+    "./coders": "./dist/src/apache_beam/coders/index.js",
```

Comment:
Have you found these to be explicitly needed?

Reply:
yes

Reply:
I'm curious, what for?

Reply:
I was playing the coders, is that not something I should be doing?

Reply:
I'm hoping that most of the time users won't have to explicitly specify or otherwise concern themselves with coders. But I could be wrong.